provider/resources.go

```diff
@@ -123,51 +123,51 @@ func Provider() tfbridge.ProviderInfo {
 		// PluginDownloadURL is an optional URL used to download the Provider
 		// for use in Pulumi programs
 		// e.g https://github.com/org/pulumi-provider-name/releases/
-		PluginDownloadURL: "",
+		PluginDownloadURL: "https://github.com/Displayr/pulumi-signalfx-synthetics/releases/download/v0.0.12/",
```

Comment:
specify the version here to work with Pulumi's custom publish script https://github.com/jaxxstorm/scripts/tree/third_party/ci